android/src/main/java/de/simolus3/fluttie/FluttiePlugin.java

```diff
@@ -12,31 +13,26 @@
 import com.airbnb.lottie.LottieCompositionFactory;
 import com.airbnb.lottie.LottieListener;
 
-import org.json.JSONException;
-import org.json.JSONObject;
-
 import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.atomic.AtomicInteger;
 
 import androidx.annotation.Nullable;
-import io.flutter.plugin.common.EventChannel;
 import io.flutter.plugin.common.MethodChannel;
 import io.flutter.plugin.common.MethodChannel.MethodCallHandler;
 import io.flutter.plugin.common.MethodChannel.Result;
 import io.flutter.plugin.common.MethodCall;
 import io.flutter.plugin.common.PluginRegistry.Registrar;
 import io.flutter.view.TextureRegistry;
 
-public class FluttiePlugin implements MethodCallHandler, EventChannel.StreamHandler, Application.ActivityLifecycleCallbacks {
+public class FluttiePlugin implements MethodCallHandler, Application.ActivityLifecycleCallbacks {
```

Comment:
Thanks for migrating the usage of an event stream to the regular async method call API here, makes it much easier to understand what's going on :+1: